src/include/simeng/memory/MemPacket.hh

```diff
@@ -87,13 +89,13 @@ class MemPacket {
   }
 
   /** Static function used to create a read request. */
-  static std::unique_ptr<MemPacket> createReadRequest(uint64_t vaddr,
-                                                      uint64_t size,
+  static std::unique_ptr<MemPacket> createReadRequest(uint64_t address,
+                                                      uint16_t size,
                                                       uint64_t reqId);
 
   /** Static function used to create a write request. */
-  static std::unique_ptr<MemPacket> createWriteRequest(uint64_t vaddr,
-                                                       uint64_t size,
+  static std::unique_ptr<MemPacket> createWriteRequest(uint64_t address,
+                                                       uint16_t size,
```

Comment:
Sould probably specify if paddr or vaddr for createRequest

Reply:
done